common/src/main/java/com/radixdlt/monitoring/Metrics.java

```diff
@@ -249,10 +248,7 @@ public record LedgerSync(
       Gauge currentStateVersion,
       Gauge targetStateVersion) {}
 
-  public record Mempool(Timer addTransaction) {}
-
-  public record V1Mempool(
-      Gauge size, Counter relaysSent, Counter addSuccesses, Counter addFailures) {}
+  public record Mempool(Counter relaysSent) {}
```

Comment:
_(Note from author)_

This was previously mis-grouped.
The entire "v1" was not used anymore (not even read by tests), so I removed it.